cli/cmd/install.go

```diff
@@ -181,6 +99,9 @@ func init() {
 
 	installParams.Namespace = installCmd.Flags().StringP("namespace", "n", "keptn",
 		"Specify the namespace where Keptn should be installed in (default keptn).")
+
+	installParams.ValuesFile = installCmd.Flags().StringP("values", "f", "",
+		"Specify values in a YAML file or a URL.")
```

Comment:
If we want to make this more like helm, maybe we can allow the possibility of multiple values files, or additionally setting values from the command line using `--set` 

[source](https://github.com/helm/helm/blob/66034e403548417ac90d9769f5dcde2d354e0033/cmd/helm/flags.go#L41-L46)

Reply:
I think providing the option to pass values using `--set` is a good idea. @christian-kreuzberger-dtx @agrimmer what do you think?

Reply:
Sounds good

Reply:
Updated :)